src/config/securityHeaders.ts

```diff
@@ -12,7 +12,7 @@ import { CYPRESS_MNEMONIC, IS_PRODUCTION } from '@/config/constants'
 export const ContentSecurityPolicy = `
  default-src 'self';
  connect-src 'self' *;
- script-src 'self' https://www.google-analytics.com https://ssl.google-analytics.com 'unsafe-inline' https://*.getbeamer.com https://www.googletagmanager.com ${
+ script-src 'self' https://www.google-analytics.com https://ssl.google-analytics.com 'unsafe-inline' https://*.getbeamer.com https://www.googletagmanager.com https://*.ingest.sentry.io ${
```

Comment:
Are all Sentry logs using the `*.ingest` subdomain or should we allow all?

Reply:
Good point. Should I change to `https://*.sentry.io` ?

Reply:
I was trying to not be so loose with it

Reply:
I glossed over their docs and it seems that `*.ingest` is correct.

Reply:
I think I also need to add `https://sentry.io` given in `https://web-core.pages.dev/` the domain is different.

<img width="793" alt="Screenshot 2022-10-07 at 17 12 07" src="https://user-images.githubusercontent.com/32431609/194587121-a1851803-6859-42b5-833f-ca42992a341a.png">